tools/scripts/build/ensure-env.js

```diff
@@ -101,6 +102,15 @@ if (FREECODECAMP_NODE_ENV !== 'development') {
 } else {
   checkClientLocale();
   checkCurriculumLocale();
+  if (fs.existsSync(`${globalConfigPath}/env.json`)) {
+    const { showUpcomingChanges } = require(`${globalConfigPath}/env.json`);
+    if (env['showUpcomingChanges'] !== showUpcomingChanges) {
+      console.log(
+        'SHOW_UPCOMING_CHANGES value has changed, cleaning client cache.'
+      );
+      spawn('npm', ['run', 'clean:client']);
```

Comment:
```suggestion
      const child = spawn('npm', ['run', 'clean:client']);
      child.stdout.setEncoding('utf8');
      child.stdout.on('data', function (data) {
        console.log(data);
      });
      child.on('error', (err) => {
        console.error(err);
      });
```
This lets us see that something's happening (otherwise it freezes for a while) or broken.